server/src/test/java/io/seata/server/lock/db/DataBaseLockManagerImplTest.java

```diff
@@ -38,6 +42,8 @@
 /**
  * @author zhangsen
  */
+@SpringBootTest
```

Comment:
When I check the spring boot document I found that there is no need to add @ExtendWith(SpringExtension.class) when use junit 5 and @SpringBootTest already has it.
![image](https://user-images.githubusercontent.com/4516752/132320228-6d9a52d3-0ad1-4a2b-a103-5412417ba8ab.png)

Reply:
fixed